src/microbe_stage/Microbe.Contact.cs

```diff
@@ -568,6 +568,22 @@ public void UnbindAll()
         Colony?.RemoveFromColony(this);
     }
 
+    /// <summary>
+    ///   Updates the microbes LookAtPoint
+    /// </summary>
+    internal void UpdateLookAtPoint()
```

Comment:
This needs a better name, one that explicitly says this creates a new look at point based on the rotation this cell has in a colony.